packages/component-library/src/components/form/_internal/mt-base-field/mt-base-field.vue

```diff
@@ -38,7 +38,10 @@
 
     <slot name="error" />
 
-    <div class="mt-field__hint-wrapper">
+    <div
+      v-if="!!$slots['field-hint'] || !!$slots['field-hint-right']"
```

Comment:
The v-if with slots is critical. We shouldn't use this anymore because it won't get updated when the slot gets filled later. In the case of a form field this could be a valid case when a hint gets added later on.

I would prefer :empty in CSS